spec/requests/flows/assertion_spec.rb

```diff
@@ -50,23 +103,23 @@
   end
 
   context "with invalid user assertion" do
-    scenario "should not issue new token with bad assertion" do
+    it "should not issue new token with bad assertion" do
       expect {
         post assertion_endpoint_url( client: @client, assertion: 'i_dont_exist' )
       }.to_not change { Doorkeeper::AccessToken.count }
 
-      should_have_json 'error', 'invalid_resource_owner'
-      should_have_json 'error_description', translated_error_message(:invalid_resource_owner)
+      should_have_json 'error', 'invalid_grant'
+      should_have_json 'error_description', translated_error_message(:invalid_grant)
       expect(response.status).to eq(401)
     end
 
-    scenario "should not issue new token without assertion" do
+    it "should not issue new token without assertion" do
       expect {
         post assertion_endpoint_url( client: @client )
       }.to_not change { Doorkeeper::AccessToken.count }
 
-      should_have_json 'error', 'invalid_resource_owner'
-      should_have_json 'error_description', translated_error_message(:invalid_resource_owner)
+      should_have_json 'error', 'invalid_grant'
+      should_have_json 'error_description', translated_error_message(:invalid_grant)
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.